src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -150,7 +144,7 @@ export const SearchBarDropdown = (props: SearchBarDropdownProps) => {
         </SuspenseWithPreviousRenderAsFallback>
         {showChainComingSoonBadge && (
           <ChainComingSoonBadge>
-            <ChainLogo src={logoUri} />
+            <ChainLogo chainId={chainId} size={20} style={{ marginRight: '8px' }} />
```

Comment:
nit: Can you keep this consistent with `StyledChainLogo` in `src/components/NavBar/ChainSelectorRow.tsx` where you use a styled component over inline styles. Or vice-versa